skimage/_shared/testing.py

```diff
@@ -181,15 +181,19 @@ def mono_check(plugin, fmt='png'):
 def setup_test():
     """Default package level setup routine for skimage tests.
 
-    Import packages known to raise errors, and then
+    Import packages known to raise warnings, and then
     force warnings to raise errors.
-    Set a random seed
+
+    Also set a random seed.
```

Comment:
Set the random seed to '0'.  This sentence can easily be interpreted that the seed gets set randomly.